### DISCUSSION TITLE
Request new feature | Application carnet de vaccination covid 19

### DISCUSSION THREAD

**Discussion by ilyhel** [cds-snc/covid-alert-documentation#68] [Ideas]
# Résumé

Bientôt arriver de vaccin de covid-19 mais y a il  un processus de gestion?

Pourquoi pas une application de gestion de vaccin, prise de rendez-vous, ordre de priorité, gestion de distribution de vaccin, disponibilité en stock, rapports de situation liéé au processus de vaccination

**Comment by timarney** [comment]
cc: @sidewalkballet ☝️

**Comment by timarney** [comment]
Thanks @ilyhel 

This has been brought back to the team for further discussion. Going to close this issue and will re-open as needed.

**Comment by LeCanardQuoi** [comment]
As a user, it would make so much sense that people that made the effort of downloading and using the app would not see the vaccine queue jumped by people that did not make that effort. Vaccine info registered in the app data would help drive higher adoption rate.

**Comment by LeCanardQuoi** [comment]
Could this be moved toward the discussion area? The app adoption rates will certainly be further challenged with the vaccine rollout. Messaging likely needs to be revised. UK NHS is considering  how to handle, perhaps related to QR code scanning and accessing public spaces. https://www.dorsetecho.co.uk/news/19157179.latest-guidance-nhs-covid-19-app/
